src/Equinox.Cosmos/Equinox.Cosmos.fsproj

```diff
@@ -27,8 +27,8 @@
 
     <PackageReference Include="FsCodec.NewtonsoftJson" Version="1.0.0-pr.20.rc2.5" />
     <PackageReference Include="FSharp.Control.AsyncSeq" Version="2.0.21" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB" Version="2.0.0" Condition=" '$(TargetFramework)' != 'netstandard2.0' " />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.0.0" Condition=" '$(TargetFramework)' == 'netstandard2.0' " />
+    <PackageReference Include="Microsoft.Azure.DocumentDB" Version="2.7.0" Condition=" '$(TargetFramework)' != 'netstandard2.0' " />
```

Comment:
putting a hard 2.7 dependency in is pretty debatable for me; in general I try to be conservative and let apps do the upgrade as desired. Will hence seek to find the lowest that compiles (i.e probably 2.2.0 as you suggest) - will double check this is tenable for existing uses too